google/cloud/aiplatform/training_jobs.py

```diff
@@ -3392,24 +3392,49 @@ def _run(
 
         model = gca_model.Model(display_name=model_display_name)
 
-        return self._run_job(
+        new_model = self._run_job(
             training_task_definition=training_task_definition,
             training_task_inputs=training_task_inputs_dict,
             dataset=dataset,
             training_fraction_split=0.8,
             validation_fraction_split=0.1,
             test_fraction_split=0.1,
             predefined_split_column_name=predefined_split_column_name,
-            model=model,
+            model=model
         )
 
+        if export_evaluated_data_items:
+            _LOGGER.info("Exported examples available at:\n%s" % self.evaluated_data_items_bigquery_uri)
+
+        return new_model
+
     @property
     def _model_upload_fail_string(self) -> str:
         """Helper property for model upload failure."""
         return (
             f"Training Pipeline {self.resource_name} is not configured to upload a "
             "Model."
         )
+    
+    @property
+    def evaluated_data_items_bigquery_uri(self) -> Optional[str]:
+        """BigQuery location of exported evaluated examples from the Training Job"""
+
+        if self._gca_resource is None:
+          _LOGGER.warning("BigQuery uri for evaluated data items does not exist. Must export evaluated data items during training.")
+          return
+        
+        try:
+            meta = getattr((self._gca_resource), "training_task_metadata")
+        except ValueError:
+            _LOGGER.warning("BigQuery uri for evaluated data items does not exist. Must export evaluated data items during training.")
+            return
+
+        if meta is not None and "evaluatedDataItemsBigqueryUri" in meta._pb:
+          return meta._pb["evaluatedDataItemsBigqueryUri"].string_value
+        else:
+          _LOGGER.warning("BigQuery uri for evaluated data items does not exist. Must export evaluated data items during training.")
```

Comment:
Would we want to raise a `ValueError` here with the same text instead of logging a warning?

Reply:
Sounds good. Done.